### ISSUE TITLE
<tg-forward> is not supported

### ISSUE THREAD

**Issue by ChungZH** [Nheko-Reborn/nheko#1167]
### Describe the bug

https://github.com/mautrix/telegram/blob/121660776310d5909c416c38be61f2b9b07db475/mautrix_telegram/formatter/from_telegram.py#L125

mautrix-telegram use `<tg-forward></tg-forward>` and Nheko don't support it.

### To Reproduce

1. Use mautrix/telegram and forward a channel message
2. See error

### Expected behavior

Element can support this well: 
![image](https://user-images.githubusercontent.com/42088872/186544917-72505c40-1179-4500-a901-5c853b392642.png)

### Screenshots

![image](https://user-images.githubusercontent.com/42088872/186544233-d26cbc04-c894-4764-bd74-e087ce273bfa.png)

### Version

0.10.0-7df260f5

### Operating system

Windows

### Installation method

Windows download

### Did you use profiles?

- [ ] Profiles used?

**Comment by nico-famedly** [comment]
This isn't really part of the matrix specification, so I am reluctant to support it, but I guess proper forwards are not in the spec yet.

**Comment by deepbluev7** [comment]
This isn't really part of the matrix specification, so I am reluctant to support it, but I guess proper forwards are not in the spec yet.

**Comment by JeanPaulLucien** [comment]
@deepbluev7 I have a look on Matrix specification. It seems not finished as text, but there is an internal Emenet docs, because Bridges are Vector's bots. In my opinion Vector will not change this, because Element and bots use this feature.

**Comment by CyberTailor** [comment]
I think unknown tags should be ignored

**Comment by msrd0** [comment]
> I think unknown tags should be ignored

I disagree. Displaying the unknown tag, especially in this case, adds valuable information for me that would otherwise be lost.
